src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```diff
@@ -2915,18 +2915,19 @@ public void addAndAbortRepairs(String clusterName, String keyspace) throws Throw
             .assertThat(response.getStatus())
             .isEqualTo(Response.Status.CREATED.getStatusCode())
             .withFailMessage(responseData);
-        String id = "";
+        UUID id = UUID.randomUUID();
         try {
-          id = new ObjectMapper().readValue(responseData, String.class);
-          testContext.addCurrentRepairId(UUID.fromString(id));
+          RepairRun repairRun = new ObjectMapper().readValue(responseData, RepairRun.class);
+          id = repairRun.getId();
+          testContext.addCurrentRepairId(id);
```

Comment:
I checked the RepairRun resource and how we deserialize responses from the API in the tests and understood what's wrong here.

A POST operation to the "/repair_run" endpoint will return a RepairRunStatus object.
To deserialize it, you'll have to do the same as is done on line 1174:

```
RepairRunStatus run = SimpleReaperClient.parseRepairRunStatusJSON(responseData);
```

then you can access `run.getId()`.

```
          RepairRunStatus run = SimpleReaperClient.parseRepairRunStatusJSON(responseData);
          id = run.getId();
          ...
```

Reply:
Thanks for clarifying, I thought it was a RepairRun object... I've made the required changes and serialisation now appears to work OK!